cli/src/main/java/org/apache/falcon/cli/FalconCLI.java

```diff
@@ -48,7 +48,45 @@
     public static final AtomicReference<PrintStream> ERR = new AtomicReference<PrintStream>(System.err);
     public static final AtomicReference<PrintStream> OUT = new AtomicReference<PrintStream>(System.out);
 
-    private static final String FALCON_URL = "FALCON_URL";
+    public static final String ENV_FALCON_DEBUG = "FALCON_DEBUG";
+    public static final String DEBUG_OPTION = "debug";
+    public static final String URL_OPTION = "url";
+    public static final String FALCON_URL = "FALCON_URL";
+
```

Comment:
SUBMIT_AND_SCHEDULE_OPT is handled twice in the merged code. Once in lines 291 to 294, again in line 302-305.